app/assets/javascripts/modal/modal-editor.js

```diff
@@ -0,0 +1,15 @@
+function ModalEditor ($module) {
+  this.$module = $module
+  if (this.$module == null) {
```

Comment:
We could avoid the utility having to know to do this check by moving where we construct it in the consuming modules. I took this approach in [my refactoring PR](https://github.com/alphagov/content-publisher/pull/1061/files#diff-68f3cc805b20202812678437322d8725R11).

Reply:
Good idea - I've got rid of these and now initialize ModalEditor in the init function for each modal JS file.